packages/client/hmi-client/src/components/workflow/tera-simulate.vue

```diff
@@ -0,0 +1,201 @@
+							class="model-configuration"
+							showGridlines
+							:value="simConfigs"
+						>
+							<ColumnGroup type="header">
+								<Row>
+									<Column selection-mode="multiple" headerStyle="width: 3rem" />
+									<Column v-for="(v, i) of simVars" :key="i" :header="v" />
+								</Row>
+							</ColumnGroup>
+							<Column selection-mode="multiple" headerStyle="width: 3rem" />
+							<Column v-for="(value, i) of simVars" :key="i" :field="value"> </Column>
+						</DataTable>
+					</AccordionTab>
+					<AccordionTab>
+						<template #header> Simulation Time Range </template>
+						<div class="sim-tspan-container">
+							<div class="sim-tspan-group">
+								<label for="1">Units</label>
+								<Dropdown
+									id="1"
+									class="p-inputtext-sm"
+									v-model="openedWorkflowNodeStore.tspanUnit"
+									:options="TspanUnitList"
+								/>
+							</div>
+							<div class="sim-tspan-group">
+								<label for="2">Start date</label>
+								<InputNumber
+									id="2"
+									class="p-inputtext-sm"
+									v-model="openedWorkflowNodeStore.tspan[0]"
+									inputId="integeronly"
+								/>
+							</div>
+							<div class="sim-tspan-group">
+								<label for="3">End date</label>
+								<InputNumber
+									id="3"
+									class="p-inputtext-sm"
+									v-model="openedWorkflowNodeStore.tspan[1]"
+									inputId="integeronly"
+								/>
+							</div>
+						</div>
+					</AccordionTab>
+				</Accordion>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+import Accordion from 'primevue/accordion';
+import AccordionTab from 'primevue/accordiontab';
+import Column from 'primevue/column';
+import Row from 'primevue/row';
+import ColumnGroup from 'primevue/columngroup';
+import DataTable from 'primevue/datatable';
+import Dropdown from 'primevue/dropdown';
+import Button from 'primevue/button';
+import InputNumber from 'primevue/inputnumber';
+
+import { useOpenedWorkflowNodeStore } from '@/stores/opened-workflow-node';
+import { TspanUnits } from '@/types/SimulateConfig';
+
+import SimulateChart from './tera-simulate-chart.vue';
+import ModelDiagram from '../models/tera-model-diagram.vue';
+
+const openedWorkflowNodeStore = useOpenedWorkflowNodeStore();
+
+enum SimulateTabs {
+	input,
+	output
+}
+
+const activeTab = ref(SimulateTabs.input);
+const node = ref(openedWorkflowNodeStore.node);
+
+const TspanUnitList = computed(() =>
+	Object.values(TspanUnits).filter((v) => Number.isNaN(Number(v)))
+);
+
+const simVars = computed(() => [
+	'Configuration Name',
+	...(node.value?.inputs[0].value as any[])[0].model.content.S.map((state) => state.sname),
+	...(node.value?.inputs[0].value as any[])[0].model.content.T.map((state) => state.tname)
+]);
+
+const simConfigs = computed(
+	() =>
+		node.value?.outputs[0]?.value?.[0].runConfigs.map(
+			(runConfig: typeof openedWorkflowNodeStore, i: number) => ({
```

Comment:
Can we group `initialValues` and `parameterValues` into a type instead of doing `typeof openedWorkflowNodeStore`?

Reply:
fixed